File: LC_Gen_Codes/2784_no_setting.py
class Solution:
    def isGood(self, nums: List[int]) -> bool:
        n = max(nums) # get the maximum element in the array
        base = [i for i in range(1,n)] + [n,n] # create the base array
        
        nums.sort() # sort the nums array
        
        return nums == base # check if nums is equal to base array

File: LC_Gen_Codes_Java/2784_no_setting.java
import java.util.Arrays;

class Solution {
    public boolean isGood(int[] nums) {
        int n = Arrays.stream(nums).max().getAsInt(); // get the maximum element in the array
        int[] base = new int[n];
        
        for (int i = 0; i < n - 1; i++) {
            base[i] = i + 1;
        }
        base[n-1] = n;
        
        Arrays.sort(nums); // sort the nums array
        
        return Arrays.equals(nums, base); // check if nums is equal to base array
    }
}